libusb/libusbi.h

```diff
@@ -30,6 +30,7 @@
 #include <inttypes.h>
 #include <stdarg.h>
 #include <stddef.h>
+#include <stdio.h>
```

Comment:
with the chanes to `configure.ac` do we still need these?

Reply:
any comments?

Reply:
yes this is still needed.